server/main.py

```diff
@@ -0,0 +1,131 @@
+import json
+import os
+import uuid
+from datetime import datetime
+from flask import Flask, request, jsonify
+
+from utilities import dir_utils
+
+DEFAULT_PORT = 8080
```

Comment:
Interesting choice, did you check that this port is not serving a specific purpose?